src/Emanote/Model/Stork/Index.hs

```diff
@@ -98,7 +113,36 @@ fileCodec =
     <*> Toml.text "url" .= fileUrl
     <*> Toml.text "title" .= fileTitle
 
+showHandling :: Handling -> Text
+showHandling handling = case handling of
+  Handling_Ignore -> "Ignore"
+  Handling_Omit -> "Omit"
+  Handling_Parse -> "Parse"
+
+parseHandling :: Text -> Either Text Handling
+parseHandling handling = case handling of
+  "Ignore" -> Right Handling_Ignore
+  "Omit" -> Right Handling_Omit
+  "Parse" -> Right Handling_Parse
+  other -> Left $ "Unsupported value for frontmatter handling: " <> other
+
+handlingCodec :: Toml.Key -> TomlCodec Handling
+handlingCodec = textBy showHandling parseHandling
```

Comment:
Is there no way to implement this codec in terms of `Show` instance?

Reply:
Tomland supports using `Show` and `Read`. They would need to be manually implemented instead of derived, so it uses `"Omit"` instead of `"Handling_Omit"`. Not much difference, but I guess it's the correct way.

Reply:
If we can implement Aeson instance generically, perhaps it would be simpler to just derive the Tomland codec via that Aeson instance. No need to modify the Show/Read instance. 

Just an idea; I haven't looked deep into how Tomland does things. You could also try asking in https://github.com/kowainik/tomland for idiomatic ideas ...